wdl/tasks/stampipes.wdl

```diff
@@ -168,3 +168,30 @@ task cutfragments {
         disks: resources.disks
     }
 }
+
+
+task chrom_buckets {
+    input {
+        File fai
+        Resources resources
+        String genome_name
+    }
+
+    command {
+        make all -f $(which chrombuckets.mk) \
+            FAI=~{fai} \
+            GENOME=~{genome_name} \
+            BUCKETS_DIR=.
+    }
+
+    output {
+        File chrombuckets = "chrom-buckets." + genome_name + ".75_20.bed.starch"
```

Comment:
We should maybe make the 
```
WIN ?= 75
BINI ?= 20
``` 
params explicit in the make command, and set the in input (bind 75/20 in the subworkflow that calls this task). I doubt these will ever change but you can see https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/98a5792859d0f92427d4c37ddc46328a8ebe9f1c/wdl/tasks/awk.wdl#L38-L39 for how the window and bin variables pop up other places.

Reply:
Wondering if this should be just set in the task level. As you said, this probably never changes because it is hardcoded in the makefile?

Reply:
hmm, actually it is probably a good idea to have them visible on the subworkflow as well.

Reply:
Task should be generic, subworkflow should be where we bind specific params we're using for our pipeline.